src/Nethermind/Nethermind.TxPool/Filters/FeeTooLowFilter.cs

```diff
@@ -18,20 +19,29 @@ internal sealed class FeeTooLowFilter : IIncomingTxFilter
         private readonly IChainHeadSpecProvider _specProvider;
         private readonly IChainHeadInfoProvider _headInfo;
         private readonly TxDistinctSortedPool _txs;
+        private readonly TxDistinctSortedPool _blobTxs;
         private readonly bool _thereIsPriorityContract;
         private readonly ILogger _logger;
 
-        public FeeTooLowFilter(IChainHeadInfoProvider headInfo, TxDistinctSortedPool txs, bool thereIsPriorityContract, ILogger logger)
+        public FeeTooLowFilter(IChainHeadInfoProvider headInfo, TxDistinctSortedPool txs, TxDistinctSortedPool blobTxs, bool thereIsPriorityContract, ILogger logger)
         {
             _specProvider = headInfo.SpecProvider;
             _headInfo = headInfo;
             _txs = txs;
+            _blobTxs = blobTxs;
             _thereIsPriorityContract = thereIsPriorityContract;
             _logger = logger;
         }
 
         public AcceptTxResult Accept(Transaction tx, TxFilteringState state, TxHandlingOptions handlingOptions)
         {
+            if (tx.SupportsBlobs && tx.MaxPriorityFeePerGas < 1.GWei())
```

Comment:
`1.Gwei` hardcoded - can we at least move it to constant if not parameter?
Also why not a filter?